Files/BaseLayout.cs

```diff
@@ -376,5 +381,41 @@ protected void InitializeDrag(UIElement element)
                 }
             }
         }
+
+        // VirtualKey doesn't support / accept plus and minus by default.
+        public readonly VirtualKey plusKey = (VirtualKey)187;
```

Comment:
What about "Add" [key](https://docs.microsoft.com/en-us/uwp/api/windows.system.virtualkey?view=winrt-18362)? (code 107)

Reply:
As far as I can see, the "Add" key is the + key located on a numeric keypad. Not all keyboards have the numeric keypad, so I chose this route so that the + and - on the main keyboard are used instead.

Reply:
Will it work if you change grid size using numpad keys (+/-)?

Reply:
I've added support for the numpad keys now👍